C-Plus-Plus/dp/shortest_uncommon_subseq.cpp

```diff
@@ -0,0 +1,69 @@
+// A subsequence is a sequence that appears in the same relative order, but not necessarily contiguous.
+//  A string of length n has 2^n different possible subsequences.
+
+// EXAMPLE:- Input : S = “babab” T = “babba”
+//           Output : 3
+//           The subsequence “aab” of length 3 is present in S but not in T.
+
+#include<bits/stdc++.h>
+using namespace std;
+
+#define MAX 1005
+
+// Returns length of shortest common subsequence
+int shortestSeq(char *S, char *T)
+{
+    int m = strlen(S), n = strlen(T);
+
+    // declaring 2D array of m + 1 rows and
+    // n + 1 columns dynamically
+    int dp[m+1][n+1];
+
+    // T string is empty
+    for (int i = 0; i <= m; i++)
+        dp[i][0] = 1;
+
+    // S string is empty
+    for (int i = 0; i <= n; i++)
+        dp[0][i] = MAX;
+
+    for (int i = 1; i <= m; i++)
+    {
+        for (int j = 1; j <= n; j++)
+        {
+            char ch = S[i-1];
+            int k;
+            for (k = j-1; k >= 0; k--)
+                if (T[k] == ch)
+                    break;
+
+            // char not present in T
+            if (k == -1)
+                dp[i][j] = 1;
+            else
+               dp[i][j] = min(dp[i-1][j], dp[i-1][k] + 1);
+        }
+    }
+
+    int ans = dp[m][n];
+    if (ans >= MAX)
+        ans = -1;
+
+    return ans;
+}
+
+int main()
+{
+    char S[] = "babab";
```

Comment:
@somya51p Take input from the user it is still not updated.

Reply:
@iamrajiv By mistake, I have reverted the changes. It is fine now. Please review and pardon me for my carelessness.